src/helpers/figures/charts/combo_chart.ts

```diff
@@ -242,7 +242,48 @@ export function createComboChartRuntime(chart: ComboChart, getters: Getters): Co
   const fontColor = chartFontColor(chart.background);
   const config = getDefaultChartJsRuntime(chart, labels, fontColor, localeFormat);
   const legend: DeepPartial<LegendOptions<"bar">> = {
```

Comment:
`bar` ?

Reply:
Yes, a combo chart is a bar chart with some series set as line chart